generate-release-vars.sh

```diff
@@ -70,8 +70,8 @@ cat >|release.auto.pkrvars.hcl <<EOF
 ami_version          = "$ami_version"
 ecs_agent_version    = "$agent_version"
 ecs_init_rev         = "$ecs_init_rev"
-docker_version       = "20.10.13"
-containerd_version   = "1.4.13"
+docker_version       = "20.10.17"
+containerd_version   = "1.6.6"
```

Comment:
Do we want to mention this in the changelog as well

Reply:
we mentioned it in the last release's changelog entry (`enhancement: Updating docker verison to 20.10.17 and containerd version to 1.6.6 for AL2`) but didn't update the generation script.